lib/dealership.rb

```diff
@@ -0,0 +1,14 @@
+class Dealership
+  attr_reader :dealership, :address, :inventory,
+  :inventory_count
+  def initialize(dealership, address)
+    @dealership = dealership
```

Comment:
No need to initialize the same class in this way; you don't use `@dealership` in this code anywhere, so it's not needed.